main/zoe/api/zoe-contract-facet.md

```diff
@@ -267,7 +268,33 @@ to manipulate the offer. The queries and operations are as follows:
         Asset: AmountMath.make(quatloosBrand, 5n),
         Price: AmountMath.make(quatloosBrand, 9n)
       }
-      ```
+    ```
+ ### `ZCFSeat.incrementBy(amountKeywordRecord)`
+  - `amountKeywordRecord`: `{AmountKeywordRecord}``
+  - Returns: `{AmountKeyRecord}`
+  - Adds the `amountKeywordRecord` argument to the `ZCFseat`'s staged allocation and returns the
+    resulting new staged allocation value.     
+
+### `ZCFSeat.decrementBy(amountKeywordRecord)`
+  - `amountKeywordRecord`: `{AmountKeywordRecord}``
+  - Returns: `{AmountKeywordRecord}`
+  - Subtracts the `amountKeywordRecord` argument from the `ZCFseat`'s staged allocation and returns the
+    resulting new staged allocation value. The subtracted amount value cannot be greater than
+    the subtracted from amount (i.e. negative results are not allowed).
+
+### `ZCFSeat.clear()`
+  - Returns: `{void}`
+  - Deletes the `ZCFSeat`'s current staged allocation, if any.
+
+### `ZCFSeat.getStagedAllocation()`
+  - Returns: `{<Allocation>}`
+  - Gets and returns the `stagedAllocation`, wich is the allocation committed if the seat is reallocated over, if offer safety holds and rights are conserved.
+
+### `ZCFSeat.hasStagedAllocation()`
+  - Returns: `{boolean}`
+  - Returns `true` if there is a staged allocation, i.e. whether `incrementBy()` or `decrementBy()` has been called and `clear()` has not. Otherwise returns 
```

Comment:
```suggestion
  - Returns `true` if there is a staged allocation, i.e. whether `incrementBy()` or `decrementBy()` has been called and `clear()`  and `reallocate` have not. Otherwise returns 
```